config/seckit.settings.yml

```diff
@@ -0,0 +1,56 @@
+seckit_xss:
+  csp:
+    checkbox: true
+    vendor-prefix:
+      x: false
+      webkit: false
+    report-only: true
+    default-src: ''
+    script-src: '''self'' ''unsafe-inline'' https://fonts.googleapis.com https://www.gstatic.com https://www.google.com https://www.googletagmanager.com https://www.google-analytics.com https://cdn.jsdelivr.net'
+    object-src: ''
+    img-src: '''self'' data:'
+    media-src: ''
+    frame-src: ''
+    frame-ancestors: ''
+    child-src: ''
+    font-src: 'https://fonts.gstatic.com'
```

Comment:
Same here, or perhaps it's my local environment that checks the current page protocol and imitates it via the protocol-free URL syntax: `//fonts.gstatic.com`

In this case we should also patch those in the CD itself or otherwise force HTTPS. No need anything else noadays.